src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -128,17 +126,11 @@ export class RelayFeeCalculator {
       this.feeLimitPercent >= 0 && this.feeLimitPercent <= 100,
       "feeLimitPercent must be between 0 and 100 percent"
     );
-    assert(
-      this.capitalCostsPercent >= 0 && this.capitalCostsPercent <= 100,
-      "capitalCostsPercent must be between 0 and 100 percent"
+    this.capitalCostsConfig = Object.fromEntries(
```

Comment:
Shouldn't we also be erroring if the `capitalCostsConfig` is undefined or has no keys?